googletest/include/gtest/internal/gtest-port.h

```diff
@@ -2505,6 +2506,7 @@ typedef TypeWithSize<4>::Int Int32;
 typedef TypeWithSize<4>::UInt UInt32;
 typedef TypeWithSize<8>::Int Int64;
 typedef TypeWithSize<8>::UInt UInt64;
+typedef TypeWithSize<sizeof(void*)>::UInt UIntPtr;
```

Comment:
`IntPtr` wouldn't be used yet, but it'd be easy enough to add it for completeness. Do you prefer to include it?